examples/computer_vision/deformabledetr_coco_pytorch/data.py

```diff
@@ -0,0 +1,123 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+"""
+COCO dataset which returns image_id for evaluation.
+Mostly copy-paste from https://github.com/pytorch/vision/blob/13b35ff/references/detection/coco_utils.py
+"""
+import os
+from io import BytesIO
+
+from google.cloud import storage
+from determined.util import download_gcs_blob_with_backoff
+
+import torchvision
+from PIL import Image
+
+from ddetr.util.misc import nested_tensor_from_tensor_list
+from ddetr.datasets.coco import ConvertCocoPolysToMask, make_coco_transforms
+
+
+def unwrap_collate_fn(batch):
+    batch = list(zip(*batch))
+    return batch
+
+
+class GCSBackend:
+    def __init__(self, bucket_name):
+        self._storage_client = storage.Client(project="determined-ai")
```

Comment:
I'm assuming this is just the storage client initialization and the project field naming here is arbitrary. Totally fine, just wanted to make sure my understanding is correct.

Reply:
yep, i removed this since it will work without project if running on a determined gcloud cluster.